packages/web-components/src/components/rux-tooltip/rux-tooltip.tsx

```diff
@@ -0,0 +1,286 @@
+import {
+    Component,
+    Host,
+    h,
+    Prop,
+    Element,
+    Event,
+    EventEmitter,
+    Watch,
+    Method,
+    State,
+    Fragment,
+} from '@stencil/core'
+import {
+    Placement,
+    computePosition,
+    offset,
+    flip,
+    autoUpdate,
+    autoPlacement,
+} from '@floating-ui/dom'
+import { hasSlot } from '../../utils/utils'
+
+/**
+ * @slot (default) - The trigger for the rux-tooltip
+ * @part container - The container of the rux-tooltip text
+ * @part trigger-container - the container of the tooltip trigger
+ */
+
+export declare type ExtendedPlacement = Placement | 'auto'
```

Comment:
@micahjones13 since Placement is an import from floating-ui and thus trying to export that whole line as is breaks it, what would the best way to do this be? import Placement into the commonTypes.module?

Reply:
Yeah, import Placement into the commonTypes and then export ExtendedPlacement from there.

Reply:
All set!